'Getting Started' Issue - Cannot find module 'dashify'

Hey there,

After reading [this article](https://increasinglyfunctional.com/2022/11/07/clojurescript-react-native-2022.html) I decided to give this template a shot.

After some fiddling I have become stuck when running `nbb post_install.cljs`, which cannot find the 'dashify' library.

```cljs
(ns post-install
  (:require ["dashify$default" :as dashify]
            ["fs" :refer [readFileSync writeFileSync mkdirSync rmSync]]
            [cljs.pprint :refer [pprint]]
            [clojure.string :as str]))
```

...I can't see this library in the `package.json` either.

**Here's what I did:**

```bash
rcf@IBM01 React_Native % which node
/Users/rcf/.nvm/versions/node/v22.11.0/bin/node

rcf@IBM01 React_Native % node -v
v22.11.0

rcf@IBM01 React_Native % yarn -v
1.22.22

rcf@IBM01 React_Native % npm i -g nbb yarn dashify

changed 4 packages in 1s

1 package is looking for funding
  run `npm fund` for details
rcf@IBM01 React_Native % npx react-native init TestReactCLJS1 --template react-native-template-cljs-krell-storybook

⚠️ The `init` command is deprecated.
The behavior will be changed on 2024-12-30 (43 days).

- Switch to npx @react-native-community/cli init for the identical behavior.
- Refer to the documentation for information about alternative tools: https://reactnative.dev/docs/getting-started

Running: npx @react-native-community/cli init

                                                          
               ######                ######               
             ###     ####        ####     ###             
            ##          ###    ###          ##            
            ##             ####             ##            
            ##             ####             ##            
            ##           ##    ##           ##            
            ##         ###      ###         ##            
             ##  ########################  ##             
          ######    ###            ###    ######          
      ###     ##    ##              ##    ##     ###      
   ###         ## ###      ####      ### ##         ###   
  ##           ####      ########      ####           ##  
 ##             ###     ##########     ###             ## 
  ##           ####      ########      ####           ##  
   ###         ## ###      ####      ### ##         ###   
      ###     ##    ##              ##    ##     ###      
          ######    ###            ###    ######          
             ##  ########################  ##             
            ##         ###      ###         ##            
            ##           ##    ##           ##            
            ##             ####             ##            
            ##             ####             ##            
            ##          ###    ###          ##            
             ###     ####        ####     ###             
               ######                ######               
                                                          

              Welcome to React Native 0.76.2!                
                 Learn once, write anywhere               

✔ Downloading template
✔ Copying template
✔ Processing template
ℹ Executing post init script 
⠧ Installing dependencies

error An unexpected error occurred: Command failed with exit code 1: npm install
npm warn deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
npm warn deprecated @babel/plugin-proposal-nullish-coalescing-operator@7.18.6: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-nullish-coalescing-operator instead.
npm warn deprecated @babel/plugin-proposal-class-properties@7.18.6: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-class-properties instead.
npm warn deprecated rimraf@2.6.3: Rimraf versions prior to v4 are no longer supported
npm warn deprecated @humanwhocodes/config-array@0.13.0: Use @eslint/config-array instead
npm warn deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported
npm warn deprecated @babel/plugin-proposal-optional-chaining@7.21.0: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-optional-chaining instead.
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated @humanwhocodes/object-schema@2.0.3: Use @eslint/object-schema instead
npm warn deprecated eslint@8.57.1: This version is no longer supported. Please see https://eslint.org/version-support for other options.

----- Error --------------------------------------
Message:  Cannot find module 'dashify'
Require stack:
- /Users/rcf/Documents/React_Native/TestReactCLJS1/post_install.cljs
(node:54140) [DEP0180] DeprecationWarning: fs.Stats constructor is deprecated.
(Use `node --trace-deprecation ...` to show where the warning was created)
Cannot find module 'dashify'
Require stack:
- /Users/rcf/Documents/React_Native/TestReactCLJS1/post_install.cljs
npm error code 1
npm error path /Users/rcf/Documents/React_Native/TestReactCLJS1
npm error command failed
npm error command sh -c nbb post_install.cljs && yarn && bundle && npx pod-install
npm error A complete log of this run can be found in: /Users/rcf/.npm/_logs/2024-11-18T18_38_24_290Z-debug-0.log

> TestReactCLJS1@0.0.1 postinstall
> nbb post_install.cljs && yarn && bundle && npx pod-install
.
rcf@IBM01 React_Native % 

```

--- a/template/package.json
+++ b/template/package.json
@@ -49,7 +49,6 @@
     "react-dom": "^18.3.1",
     "react-native-gesture-handler": "^2.20.2",
     "react-native-reanimated": "^3.16.1",
-    "react-native-tcp-socket": "^6.2.0",
     "react-native-safe-area-context": "^4.14.0",
     "react-native-svg": "^15.8.0",
     "react-test-renderer": "18.3.1",

--- a/template/package.json
+++ b/template/package.json
@@ -49,6 +49,7 @@
     "react-dom": "^18.3.1",
     "react-native-gesture-handler": "^2.20.2",
     "react-native-reanimated": "^3.16.1",
+    "react-native-tcp-socket": "^6.2.0",
     "react-native-safe-area-context": "^4.14.0",
     "react-native-svg": "^15.8.0",
     "react-test-renderer": "18.3.1",

--- a/template/package.json
+++ b/template/package.json
@@ -44,6 +44,7 @@
     "babel-loader": "^8.4.1",
     "eslint": "^8.19.0",
     "jest": "^29.6.3",
+    "dashify": "^2.0.0",
     "prettier": "2.8.8",
     "react-dom": "^18.3.1",
     "react-native-gesture-handler": "^2.20.2",

